Chapters/5_Core_identifiers.md

```diff
@@ -40,39 +40,31 @@ As an example, `swh:1:cnt:94a9ed024d3859793618152ea559a168bbcbb5e2` is the SWHID
 
 Directories are data structures commonly used in hierarchical file systems to group together files and other directories, and to hold relevant metadata about them, in the form of directory entries.
 
-This version of the SWHID standard adopts the same convention as the popular *git* version control system, and only takes into account as metadata the name of the directory entries and a simplified representation of the access rights.
+This version of the SWHID standard adopts the same convention as the popular *git* version control system, and only takes into account as metadata the name of the directory entries (as a sequence of arbirary bytes, excluding ASCII '/' and the NULL byte) and a simplified representation of the access rights.
```

Comment:
Typo: "arbirary".

Reply:
@olasd may you fix this typo and update the pull request?